draft-ietf-tls-dtls-rrc.md

```diff
@@ -70,6 +70,11 @@ CID-address binding is updated in that endpoint's session state.
 This is done in order to give the receiving endpoint confidence
 that the sending peer is in fact reachable at the source address (and port) indicated in the received datagram.
 
+{{regular}} of this document explains the fundamental mechanism that aims to reduce the DDoS attack surface.
+Additionally, in {{enhanced}}, a more advanced address validation mechanism is discussed.
+This mechanism is designed to counteract off-path attackers who try to place themselves on-path by racing packets that trigger address rebinding at the receiver.
```

Comment:
Just as proposal:

`s/who try/trying/`

Reply:
done in [c91df60](https://github.com/tlswg/dtls-rrc/pull/68/commits/c91df609f8a94508ec783d21099b99b78c385e17)